packages/collector/test/tracing/cloud/aws/sqs/receiveMessage.js

```diff
@@ -0,0 +1,205 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+const instana = require('../../../../../')();
+const express = require('express');
+const port = process.env.APP_RECEIVER_PORT || 3216;
+const agentPort = process.env.INSTANA_AGENT_PORT;
+const request = require('request-promise');
+
+const app = express();
+
+const delay = require('../../../../../../core/test/test_util/delay');
+const { sqs } = require('./sqsUtil');
+const queueURL = process.env.AWS_SQS_QUEUE_URL;
+const logPrefix = `AWS SQS Message Receiver (${process.pid}):\t`;
+
+let hasStartedPolling = false;
+
+const receiveParams = {
+  AttributeNames: ['SentTimestamp'],
+  MaxNumberOfMessages: 1,
+  MessageAttributeNames: ['All'],
+  QueueUrl: queueURL,
+  VisibilityTimeout: 5,
+  WaitTimeSeconds: 5
+};
+
+function log() {
+  /* eslint-disable no-console */
+  const args = Array.prototype.slice.call(arguments);
+  args[0] = `${logPrefix}${args[0]}`;
+  console.log.apply(console, args);
+}
+
+app.get('/', (_req, res) => {
+  if (hasStartedPolling) {
+    res.send('OK');
+  } else {
+    res.status(500).send('Not ready yet.');
+  }
+});
+
+app.listen(port, () => {
+  `Receiver server started at port ${port}`;
+});
+
+function receivePromise() {
+  let span;
+  return sqs
+    .receiveMessage(receiveParams)
+    .promise()
+    .then(data => {
+      if (!data || !data.Messages || data.Messages.length === 0) {
```

Comment:
Can `data` contain something like `data.error`? I think I saw something like that in the instrumentation. We should handle that case here. Maybe just writing the error to the console, so it doesn't go unnoticed.